.github/workflows/CI.yml

```diff
@@ -1,6 +1,6 @@
 name: QUnit IE
 
-on: [push]
+on: [push, pull_request]
```

Comment:
Change doesn't look to be related to exceljs version change.
Do we need it?

Reply:
No. But we saw problems with the new ExcelJS in IE.